src/CoreWCF.ConfigurationManager/src/CoreWCF/Configuration/IStandardBindingElement.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using CoreWCF.Channels;
+
+namespace CoreWCF.Configuration
+{
+    public interface IStandardBindingElement
+    {
```

Comment:
Can you explain the reason why you introduced this interface instead of using StandardBindingElement?

Reply:
I can actually remove that. It was initially done because I had inherited from ServiceModelConfigurationElementCollection.  This created issues that were not resolvable.  I will switch everything out.

Reply:
It is needed to have the behavior of the CustomElement behave consistent with the StandardBindingElement. for the proper function of ConfigurationManagerServiceModelOptions.AddBinding().  I thought I could remove it but it would add functionality to ServiceModelExtensionCollectionElement that doesn't seem consistent with it's use cases.   If preferred I can remove the interface and add an overload specific to the CustomBinding.